rolling/release-build.yaml

```diff
@@ -0,0 +1,94 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+
+build_environment_variables:
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+
+jenkins_binary_job_priority: 78
```

Comment:
The priorities should be different from Foxy.

Same below.

Reply:
We should set the rolling distro to lower priority than all the other distros. If it's a little slower rebuilding that's ok. We don't want to worry about gating rolling releases if we're in a push for a stable fixup.

Reply:
I would say the opposite of also true: we don't want a blocked rolling rebuild lead to failures on PRs.

Reply:
My first instinct was also to de-prioritize the rolling distribution in favor of stable ones. Which is what I did in e3d9fe6.

Reply:
Maybe either of you can clarify your rational. I would think we want changes in the rolling distro to turn around asap since that is where the majority of development will happen so a quick response time on PR is important.

Reply:
> Maybe either of you can clarify your rational.

I think we might have different primary audiences in mind. I'm thinking mostly of the fact that community-wide, the stable distributions are more important and are at a higher support tier, therefore have priority.

I almost made the rolling PR jobs higher priority than stable PR jobs but it would require juggling a lot of the existing priorities and I am more focused on getting the distro bootstrapped than on settling the running priorities right now.

Reply:
> I'm thinking mostly of the fact that community-wide, the stable distributions are more important and are at a higher support tier, therefore have priority.

I don't see how the user perspective priority is relevant here. Binary packages for released distros happen whenever the maintainer does them and should sit in the testing repo for some time for adventures users to try them before they are being synced to the main repo. So I would argue if it takes a day longer to get into testing that isn't a big deal since we shouldn't sync right to main anyway right after a rebuild.

> I am more focused on getting the distro bootstrapped than on settling the running priorities right now.

Please feel free to merge this as-is to get started with `rolling` but make sure that we are following up on this topic and persist the rational and conclusion somewhere (and if necessary update priorities is they change).

Reply:
> So I would argue if it takes a day longer to get into testing that isn't a big deal since we shouldn't sync right to main anyway after a rebuild.

This is most often the case however it's not uncommon for releases which resolve known regressions to go in just before a sync during the freeze window.

> make sure that we are following up on this topic and persist the rational and conclusion somewhere 

That's reasonable to me. @tfoote do you want to elaborate on your thought process and I'll start incorporating this into documentation of the status quo which we can use as a basis for PRs with motivated changes.